src/allencell_ml_segmenter/thresholding/thresholding_view.py

```diff
@@ -326,7 +327,7 @@ def doWork(self) -> None:
         self._thresholding_model.dispatch_save_thresholded_images()
 
     def focus_changed(self) -> None:
-        self._file_input_widget._update_layer_list()
+        self._prediction_result_input_widget._update_layer_list()
```

Comment:
is this handling a tab change?
it seems like updates to the layer list should happen via a model event.